### DISCUSSION TITLE
Will this work in a WebApplication?

### DISCUSSION THREAD

**Discussion by SSzretter** [RonSijm/RonSijm.Blazyload#1] [Q&A]
As in the default template for .net8 Blazor uses:

var builder = WebApplication.CreateBuilder(args);

**Comment by RonSijm** [comment]
Which one do you mean? The top one here?
![image](https://github.com/RonSijm/RonSijm.Blazyload/assets/337928/dfeee926-530c-49ff-9b16-71815826632b)

I haven't really looked into .net8 Blazor yet, because I'm still waiting on some infrastructure 3th parties to support it, so I can't upgrade my own projects yet

**Comment by SSzretter** [comment]
Yes, it's the Blazor Web App.
